tests/components/modbus/conftest.py

```diff
@@ -38,7 +38,7 @@ async def base_test(
     entity_domain,
     array_name_discovery,
     array_name_old_config,
-    register_words,
+    register_words_or_exception,
```

Comment:
Please do not change the test methods in a PR where you implement new functionality. In case of problems it is quite hard to determine whether it is the new functionaliy or the test changes.